ui/app/AppLayouts/Profile/views/AdvancedView.qml

```diff
@@ -536,5 +539,46 @@ SettingsContentBase {
                 close()
             }
         }
+
+        Connections {
+            target: advancedStore.advancedModule
```

Comment:
(You can ignore this comment) I know this approach is repeated all over the place in our codebase but Views shouldn't know anything about module instances.

Possible solution, AdvancedStore.qml:
```
signal enableCommunityHistoryArchiveSupportFailed
Connections {
    target: advancedModule
    function onEnableCommunityHistoryArchiveSupportFailed() {
        root.enableCommunityHistoryArchiveSupportFailed()
    }
}
```